### ISSUE TITLE
Dynamically loading options doesn't work

### ISSUE THREAD

**Issue by inexuscore** [silviomoreto/bootstrap-select#945]
Hi
I've been using bootstrap-select for a long time, just recently, I integrated this plugin in one of my Ember.js projects. I load the list of options in my route object and then render them in my template using a {{#each}} loop. when i inspect the DOM everything seems be to alright, the option elements are rendered correctly under the parent <select> and bs-select is initialized properly. However when i open the selectpicker nothing shows up, as if the list of options is empty.

I tried several things like .selectpicker("render") and .selectpicker("refresh") but nothing seems to work. any ideas?

Thanks

**Comment by caseyjhol** [comment]
I'm not very familiar with Ember, but see here: http://stackoverflow.com/questions/16566700/integrating-bootstrap-select-to-work-with-ember.

**Comment by inexuscore** [comment]
Hey @caseyjhol 
I've seen that SO question, didn't help my case but thanks anyway. I refactored my code a little bit and got it working. the problem was that my data was being retrieved AFTER the selectpicker initialization. I wanted to see how i can refresh/re-render the component after such changes. I tried various things but got no results.
Anyway, its working now but I'll definitely come back and update this post if i figured it out :)

Thanks for your time
